src/components/home-page/home-page.jsx

```diff
@@ -4,23 +4,24 @@ import styled from 'styled-components';
 import { media } from '../../utils/css-utils';
 
 import SubscriptionForm from './subscription-form';
+import SocialLinks from './social-links-block';
 
 const EmptyBlock = styled.section`
-    font-family: Rubik;
-    font-size: 48px;
+    font-family: Rubik, sans-serif;
+    font-size: 4.8em;
```

Comment:
Nope we should use em/rem (here is why https://css-tricks.com/why-ems/)
And please use them in your component too.